snuba/cli/devserver.py

```diff
@@ -34,6 +34,7 @@ def devserver(bootstrap, workers):
         os.execvp(daemons[0][1][0], daemons[0][1])
 
     daemons += [
+        ('transaction-consumer', ['snuba', 'consumer', '--auto-offset-reset=latest', '--log-level=debug', '--dataset=transactions', '--consumer-group=g2']),
```

Comment:
just a hint, let's give the consumer group a meaningful name. I was using g1 and g2 during dev on my machine. What about "transacitons_group"